url/url-tojson.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>URL's toJSON()</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id=log></div>
+<script>
+test(() => {
+  const a = new URL("https://example.com/")
+  assert_equals(JSON.stringify(a), "\"" + a.href + "\"")
```

Comment:
I usually prefer having the full string in the expectation so that there is no ambiguity.